messaginginapp/src/test/java/io/customer/messaginginapp/state/InAppMessageReducerTest.kt

```diff
@@ -0,0 +1,282 @@
+package io.customer.messaginginapp.state
+
+import io.customer.commontest.config.TestConfig
+import io.customer.commontest.config.testConfigurationDefault
+import io.customer.commontest.extensions.random
+import io.customer.messaginginapp.gist.GistEnvironment
+import io.customer.messaginginapp.gist.data.model.GistProperties
+import io.customer.messaginginapp.gist.data.model.Message
+import io.customer.messaginginapp.gist.data.model.MessagePosition
+import io.customer.messaginginapp.testutils.core.JUnitTest
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+
+class InAppMessageReducerTest : JUnitTest() {
+
+    private lateinit var initialState: InAppMessagingState
+
+    override fun setup(testConfig: TestConfig) {
+        super.setup(
+            testConfigurationDefault {
+                diGraph {
+                    sdk { }
+                }
+            }
+        )
```

Comment:
I believe we can remove this empty config? 🤔

Reply:
removed